applications/dashboard/views/settings/registration.php

```diff
@@ -55,6 +55,7 @@
                 </tbody>
             </table>
         </li>
+        <?php if (c('Garden.Registration.ManageCaptcha', true)) : ?>
```

Comment:
Something was nagging me about this and I couldn't put my finger on what it was. Thanks.